chart/kubeapps/values.yaml

```diff
@@ -719,6 +719,9 @@ apprepository:
     ##   - myRegistryKeySecretName
     ##
     pullSecrets: []
+  ## @param apprepository.globalReposNamespaceSuffix Suffix for the namespace of global repos
+  ##
+  globalReposNamespaceSuffix: repos-global
```

Comment:
Agree with the rest of the comments in that the suffix should also include the `-` to prevent backward-incompatible changes.

Also, in order to stick close to similar names within our organization, why not go with a similar suffix as Carvel people does: `-packaging-global` ?

Reply:
I find using the word `repos` more intuitive 😄 
Is there any accepted terminology for this?